artemis-server/src/main/java/org/apache/activemq/artemis/core/paging/impl/PagingStoreFactoryDatabase.java

```diff
@@ -106,8 +105,8 @@ public PagingStoreFactoryDatabase(final DatabaseStorageConfiguration dbConf,
                                      final ScheduledExecutorService scheduledExecutor,
```

Comment:
I see that `JDBCSequentialFile directoryList` and `JDBCSequentialFileFactoryDriver dbDriver`